dashboard/config/initializers/atomic_save.rb

```diff
@@ -0,0 +1,10 @@
+# Provides `#atomic_save!` which skips the default transaction
+# wrapping behavior of `#save!`.
+#
+# Only use in cases where validations/callbacks will never
+# need to trigger any rollback of the operation.
+module ActiveRecord
+  module Persistence
+    alias_method :atomic_save!, :save!
```

Comment:
`ActiveRecord::Base` is [composed](https://github.com/rails/rails/blob/v5.0.1/activerecord/lib/active_record/base.rb#L291-L314) by including [Persistence](https://github.com/rails/rails/blob/v5.0.1/activerecord/lib/active_record/persistence.rb#L151-L153) module followed by [Transactions](https://github.com/rails/rails/blob/v5.0.1/activerecord/lib/active_record/transactions.rb#L323-L325), both of which define `#save!`. `Transactions#save!` overrides/shadows `Persistence#save!`, wrapping it in a transaction. By aliasing `Persistence#save!` we can escape the transaction-wrapping behavior, only when calling the new alias method.

It's not a critical tweak (it reduces 3 DB commands `BEGIN; INSERT [...]; COMMIT` to just the `INSERT`), but applied to our highest-load DB actions I think it might make a noticeable difference.

See [related discussion on SO](https://stackoverflow.com/q/29075767/2518355).